### PR TITLE
Refactor servers

### PR DESCRIPTION
**Pull request by digitalextremist**
Testing refactored server, per celluloid/reel#120 decision to decentralize `Server`

### CODE REVIEW

File: lib/reel/server.rb [hunk 1]
```diff
@@ -13,64 +21,19 @@ class Server
...
-    # @option options [true] spy on the request
-    #
-    def self.unix(socket_path, options = {}, &callback)
-      server = Celluloid::IO::UNIXServer.new(socket_path)
-
-      self._new(server, options, &callback)
-    end
-
-    def initialize(server, options = {}, &callback)
+    def initialize(server,options={}, &callback)
```
Comment (stouset) [inline lib/reel/server.rb:24]: Pedantic, but you're missing a space here.

**Comment by stouset** [comment]
Obviously already merged, but this looks great. Kudos for the excellent work.

**Comment by digitalextremist** [comment]
@stouset change made - this is my "testing" fork and branch, about to verify what will be the PR is perfect -- looks like a couple last things to bring into decentrality/reel which is where the PR will be from to satisfy #120

Thanks --

**Comment by stouset** [comment]
Ah, whoops. It wasn't obvious that this was in your own repo and not in `celluloid/reel`.

**Comment by stouset** [comment]
If there's anything I can do to help finish this up, let me know. Given that my solution caused failures that weren't caught by the tests, we should probably have tests for that particular type of failure.

**Comment by digitalextremist** [comment]
Posted everything into celluloid/reel now, and will resume over there on #120.

**Pull request merged**